rust/simd/Makefile

```diff
@@ -0,0 +1,44 @@
+.PHONY: all
+all: build
+
+.PHONY: build
+.SILENT: build
+build:
+	dfx canister create --all
+	dfx build
+
+.PHONY: install
+.SILENT: install
+install: build
+	dfx canister install heartbeat --argument 1
```

Comment:
I don't understand the heartbeat and timer parts here. Is this a copy from another project?